composer.json

```diff
@@ -20,9 +20,9 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
-        "symfony/symfony": "~2.3",
-        "twig/twig": ">=1.23.0"
+        "php": ">=5.3.2",
```

Comment:
@DonCallisto I agree.

The class name resolution is used in this PR. But this feature is not supported in PHP 5.3.2. 
The PHP version should be `>=5.5`